### PR TITLE
test(cli): Update CLI tests

### PR DESCRIPTION
**Pull request by ezishiri**

### CODE REVIEW

File: tests/cli/test_asm.py [hunk 1]
```diff
@@ -168,28 +174,28 @@ def test_add_seeds_from_stdin(self):
...
+                "asm",
+                "add-seeds",
+                "-i",
+                "test.json",
+            ],
+            asm_auth=True,
+        )
+        self.mocker.patch(
+            "builtins.open",
+            new_callable=mock_open,
```
Comment (thehappydinoa) [inline tests/cli/test_asm.py:194]: I think we should use self.mocker.mock_open instead

```suggestion
            new_callable= self.mocker.mock_open,
```

File: tests/cli/test_subdomains.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import json
 from io import StringIO
 from typing import Set
-from unittest.mock import patch
+from unittest import TestCase
```
Comment (grace-murphy) [inline tests/cli/test_subdomains.py:5]:
```suggestion
```

File: tests/cli/test_subdomains.py [hunk 3]
```diff
@@ -29,7 +29,7 @@ def search_callback(request):
     return (200, {}, json.dumps(resp_body))
 
 
-class CensysCliSubdomainsTest(CensysTestCase):
+class CensysCliSubdomainsTest(CensysTestCase, TestCase):
```
Comment (grace-murphy) [inline tests/cli/test_subdomains.py:32]:
```suggestion
class CensysCliSubdomainsTest(CensysTestCase):
```

File: tests/cli/test_subdomains.py (hunk 2 above)
Comment (grace-murphy) [reply tests/cli/test_subdomains.py:5]: `CensysTestCase` is actually an instance of `TestCase`

**Comment by grace-murphy** [comment]
This looks great! Just those two small changes

**Review by grace-murphy — APPROVED** [review]
Looks great!

**Pull request merged**